utils/oscap-xccdf.c

```diff
@@ -1040,6 +1045,7 @@ bool getopt_xccdf(int argc, char **argv, struct oscap_action *action)
 	// options
 		{"output",		required_argument, NULL, XCCDF_OPT_OUTPUT},
 		{"results", 		required_argument, NULL, XCCDF_OPT_RESULT_FILE},
+		{"results-stig", 	required_argument, NULL, XCCDF_OPT_RESULT_FILE_STIG},
```

Comment:
we probably should put stig-viewer there because results-stig is misleading. people might switch --results to --results-stig just because they are evaluating a stig profile

Reply:
+1

Reply:
Ok, makes sense, but not even "results-stig-viewer"?